prisma/schema.prisma

```diff
@@ -1,6 +1,8 @@
 generator client {
   provider        = "prisma-client-js"
   previewFeatures = ["cockroachdb"]
+  // The following is needed for netlify
```

Comment:
```suggestion
  // The following is needed when using deploying to netlify using `netlify deploy` locally
  // It can be removed if the deployment is configured on netlify to automatically deploy for each commit
```